eclipse-scout-core/src/tooltip/Tooltip.js

```diff
@@ -26,6 +26,7 @@ export default class Tooltip extends Widget {
     this.windowPaddingX = 10;
     this.windowPaddingY = 5;
     this.origin = null;
+    this.originProducer = null;
```

Comment:
I think we need a producer for the offset as well.

originProducer: ($anchor: JQuery) => Rectangle;
offsetProducer: (origin: Rectangle) => Point;